src/python/Publisher/PublisherMasterRucio.py

```diff
@@ -313,7 +313,7 @@ def startSlave(self, task):  # pylint: disable=too-many-branches, too-many-local
                     lfnsToPublish.append(fileDict['destination_lfn'])
             FilesInfoFromTBDInBlock[blockName] = filesInfo
 
-        logger.info(f"Prepare publish info for {len(blocksToPublish)} blocks")
+        logger.info("Prepare publish info for %s blocks", len(blocksToPublish))
```

Comment:
Did we agree sometimes ago that we will use f-string in log format?
https://stackoverflow.com/a/59027338

We (sorry for blaming everybody here) just too lazy to change pylint config.

Reply:
I would very much like to use f-strings in logging calls, but do not know how to change pylint. Maybe we need Alan ? @mapellidario